### DISCUSSION TITLE
I think it is very easly detected on Cloudflare check e,g on http://c-h-a-t-g-p-t.com'

### DISCUSSION THREAD

**Discussion by hayat-khan-space** [Kaliiiiiiiiii-Vinyzu/patchright-python#8] [General]
import asyncio
# patchright here!
from patchright.async_api import async_playwright

async def main():
    async with async_playwright() as p:
        browser = await p.chromium.launch()
        page = await browser.new_page()
        await page.goto('http://c-h-a-t-g-p-t.com')
        await page.screenshot(path=f'example-{p.chromium.name}.png')
        await browser.close()

asyncio.run(main())

**Comment by ttlns** [comment]
have you tried running it on a different IP or machine?

**Reply by hayat-khan-space** [reply]
Yes 
![example-chromium](https://github.com/user-attachments/assets/6dbae142-9f78-48c8-9018-7cad6fb3fb23)

**Comment by Vinyzu** [comment]
Website isnt available anymore.

**Reply by ttlns** [reply]
I think @hayat-khan-space meant `chatgpt.com` => detection by CF

**Reply by Vinyzu** [reply]
Just tested, patchright passes CF on `chatgpt.com`.
